release_notes/osd_latest_product_updates.adoc

```diff
@@ -484,6 +705,21 @@ image::ocp33-imagedetails2.png["Image Stream Tag Configuration"]
 
 This release adds the following improvements to networking components.
 
+[[osd-subdomain-wildcard-router]]
```

Comment:
I have a PR waiting to be merged that also provides some info, #3923. Maybe useful to provide the xref once it's been merged